### PR TITLE
Changed ether balance at Slot Header to real Ethereum balance

### PR DESCRIPTION
**Pull request by fish-sushi**

### CODE REVIEW

File: app/javascripts/components/slot/layouts/header.jsx [hunk 1]
```diff
@@ -11,7 +11,9 @@ const Header = () =>
     <NavLink exact to="/slot/make" className={styles.item} activeClassName="active">
       MAKE
     </NavLink>
-    <div className={styles.walletStatus}> Your balence : 0.02342 ETH</div>
+    <div className={styles.walletStatus}>
+      {' '}Your Balance : {rootState.get('balance')} ETH
```
Comment (parkkoon) [inline app/javascripts/components/slot/layouts/header.jsx]: {' '} is for what?

File: app/javascripts/components/slot/index.jsx [hunk 2]
```diff
@@ -1,19 +1,28 @@
...
 import styles from './slot.scss';
 import { Header, Footer } from './layouts';
 import MakeGame from '../makeGame';
 import PlaySlot from '../playSlot';
 import Status404 from '../404';
 
-const Slot = ({ match }) =>
+function mapStateToProps(appState) {
+  console.log(appState.root.get('account'), "appState.root.get('account')");
+  console.log(appState.root.get('balance'), "appState.root.get('balance')");
```
Comment (parkkoon) [inline app/javascripts/components/slot/index.jsx]: Please delete console.log

**Review by parkkoon — CHANGES_REQUESTED** [review]

**Review by c0smic0wl — APPROVED** [review]
🦉

**Review by parkkoon — APPROVED** [review]
👍

**Pull request merged**